packages/vector_graphics/test/render_vector_graphics_test.dart

```diff
@@ -195,3 +222,41 @@ class FakePaintingContext extends Fake implements PaintingContext {
   @override
   final FakeCanvas canvas = FakeCanvas();
 }
+
+class FixedOpacityAnimation extends Animation<double> {
```

Comment:
Use https://api.flutter.dev/flutter/animation/AlwaysStoppedAnimation-class.html

Reply:
TIL, thanks!

Reply:
actually, I want this class for the test so I can verify the listeners

Reply:
Fair